rotkehlchen/tasks/manager.py

```diff
@@ -549,6 +581,7 @@ def _maybe_update_snapshot_balances(self) -> Optional[list[gevent.Greenlet]]:
                     save_despite_errors=False,
                     timestamp=None,
                     ignore_cache=True,
+                    detect_tokens=detect_tokens if len(detect_tokens) else None,
```

Comment:
Hmm I think this is silly. You are using the events and not really utilizing the info you get and then you do the extremely heavy token detection task. This is wrong.